CHANGELOG.md

```diff
@@ -2,13 +2,18 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+[1.44.0] - 2021-09-02
+------------------------
+##### Changed
+- Upgraded node, npm, and other dependencies
+
 [1.43.0] - 2021-05-11
 ------------------------
 ##### Added
 - Support for `odk:setgeopoint`.
 
 ##### Changed
-- changed form id attribute to data-form-id to support spaces.
+- Changed form id attribute to data-form-id to support spaces.
```

Comment:
Not necessary now but it'd be nice if we linted Markdown as part of CI/available as part of the dev environment. A lot of stuff like this can be fussy to catch manually but easy to catch/fix automatically and can help avoid commit/blame noise in the process.

Reply:
Filed #122 to track that.